tfhe/src/integer/wopbs/mod.rs

```diff
@@ -436,13 +436,13 @@ impl WopbsKey {
     /// # Example
     /// ```rust
     /// use tfhe::integer::gen_keys_crt;
-    /// use tfhe::integer::parameters::PARAM_4_BITS_5_BLOCKS;
     /// use tfhe::integer::wopbs::WopbsKey;
+    /// use tfhe::shortint::parameters::parameters_wopbs_message_carry::WOPBS_PARAM_MESSAGE_4_CARRY_4_KS_PBS;
     ///
     /// let basis: Vec<u64> = vec![9, 11];
     /// let msg_space: u64 = basis.iter().copied().product();
     ///
-    /// let param = PARAM_4_BITS_5_BLOCKS;
+    /// let param = WOPBS_PARAM_MESSAGE_4_CARRY_4_KS_PBS;
```

Comment:
Unrelated to this PR, but are the parameters `WOPBS_PARAM_MESSAGE_4_CARRY_4_KS_PBS` optimized for radix wop (without carry I guess)? Is there any advantage of doing that instead of traditional PBS?
In my view Wop is more adapted for CRT representation

Reply:
I'm just updating the docstrings here, those questions will be tackled during the wop rework, large look ups for radix had some potential advantages through wop but as there was no real alternative it's hard to say